src/Notifications/NotificationCenterController.cs

```diff
@@ -72,17 +79,56 @@ internal NotificationCenterController(DynamoView view, DynamoLogger dynLogger)
                 HorizontalOffset = notificationPopupHorizontalOffset,
                 VerticalOffset = notificationPopupVerticalOffset
             };
-            notificationUIPopup.webView.EnsureCoreWebView2Async();
-            notificationUIPopup.webView.CoreWebView2InitializationCompleted += WebView_CoreWebView2InitializationCompleted;
             logger = dynLogger;
             // If user turns on the feature, they will need to restart Dynamo to see the count
             // This ensures no network traffic when Notification center feature is turned off
             if (dynamoViewModel.PreferenceSettings.EnableNotificationCenter) 
             {
                 RequestNotifications();
+                InitializeBrowserAsync();
```

Comment:
@RobertGlobant20 @mjkkirschner I think this line is not meant to be here? With this, the constructor will always try to initialize WebView2